samples-aspnet/Samples.AspNetMvc5_0/Integrations/StackExchangeRedis.cs

```diff
@@ -0,0 +1,37 @@
+using System;
+using System.Linq;
+using StackExchange.Redis;
+
+namespace Samples.AspNetMvc5_0.Controllers
+{
+    public class StackExchangeRedis
```

Comment:
What is this file? This seems unrelated to the AAS work

Reply:
bad rebase